packages/cli/commands/create/webpack-serverless.js

```diff
@@ -6,7 +6,7 @@ module.exports = {
     cloneGitHubRepo(
       dest,
       assetType,
-      'cms-webpack-serverless-boilerplate',
+      'HubSpot/cms-webpack-serverless-boilerplate',
```

Comment:
Ended up having to change a few extra files to include `HubSpot` in the github repo URLs. Thought it was confusing that some of the github functions required it, while some didn't